Content.Server/GameObjects/Components/Disposal/DisposalMailingUnitComponent.cs

```diff
@@ -584,11 +584,7 @@ public override void ExposeData(ObjectSerializer serializer)
                 seconds => _flushDelay = TimeSpan.FromSeconds(seconds),
                 () => (int) _flushDelay.TotalSeconds);
 
-            serializer.DataReadWriteFunction(
-                "entryDelay",
-                0.5f,
-                seconds => _entryDelay = seconds,
-                () => (int) _entryDelay);
+            serializer.DataField(this, x => x._entryDelay, "entryDelay", 0.5f);
```

Comment:
You could just use `ref _entryDelay` instead of a lambda thingy right?

Reply:
It should probably be a property but I'll just change it

Reply:
Ye, both work
```suggestion
            serializer.DataField(ref _entryDelay, "entryDelay", 0.5f);
```